src/engines/gnucap.c

```diff
@@ -60,6 +60,9 @@ static struct analysis_tag analysis_tags[] = {
 #define TAGS_COUNT (sizeof(analysis_tags) / sizeof(struct analysis_tag))
 
 // Parser STATUS
+typedef enum { STATE_IDLE, IN_VARIABLES, IN_VALUES, STATE_ABORT } ParseDataState;
+static ParseDataState state;
+
```

Comment:
Why? There is no point in making this static, actually, in a multi window and multi parallel simulation case, this will break!